src/DynamoCore/DynamoCore.csproj

```diff
@@ -431,6 +427,12 @@ limitations under the License.
     <Copy SourceFiles="@(LegacyBinariesToNodesFolder)" DestinationFolder="$(OutputPath)\nodes\" />
   </Target>
   <Target Name="AfterBuild" Condition=" '$(OS)' != 'Unix' ">
+    <ItemGroup>
+      <ASMLibG Include="$(SolutionDir)..\extern\LibG_*\*" />
```

Comment:
Why is this needed? I think we are getting rid of the `extern\LibG` folders, no?

Reply:
Like I said before, under the "extern" folder during the CI pipeline execution, we are creating some folders to download nuget packages like TextTemplating and ASM that are not required for a success build, but they're needed to pass all the unit tests in case you don't have an ADSK product installed (e.g. Revit), so I just moved the logic that was there to the "After Build" like we agree on our previous talk.

Reply:
I don't think this will work on dev machine setup.

Reply:
It works fine on a dev machine 
![image](https://user-images.githubusercontent.com/47230272/91330016-7f940200-e78e-11ea-86c4-d442e5e05ddb.png)

Reply:
Ok, did you make sure you didn't have the extern\LibG folder locally?

Reply:
Yes, no LibG  folder under extern folder 
![image](https://user-images.githubusercontent.com/47230272/91333973-f1bb1580-e793-11ea-88bb-3a4ecd3a71fb.png)